openapi-spec/swagger.yaml

```diff
@@ -650,40 +766,54 @@ paths:
     put:
       tags:
         - s3
-      summary: upload object.
+      summary: Upload Object.
       operationId: objectPut
       responses:
         '200':
-          description: successful upload a object
+          description: Successful Upload Object
+        '403':
+          description: Forbidden; Access Denied, InvalidAccessKey, SignatureDoesNotMatch
+        '404':
+          description: Not Found i.e. No Such Bucket
     get:
       tags:
         - s3
-      summary: download object
+      summary: Download Object
       operationId: objectGet
       responses:
         '200':
-          description: successful download object
+          description: Successful Download Object
+        '403':
+          description: Forbidden; Access Denied, InvalidAccessKey, SignatureDoesNotMatch
+        '404':
+          description: Not Found i.e. No Such Key
     delete:
       tags:
         - s3
-      summary: delete a object.
+      summary: Delete Object.
       operationId: objectdelete
       responses:
         '200':
-          description: successful delete object
+          description: Successful Deletion of Object
           content:
             application/xml:
               schema:
                 $ref: '#/components/schemas/BaseResponse'
-  '/v1/s3/storageClasses':
+        '403':
+          description: Forbidden; Access Denied, InvalidAccessKey, SignatureDoesNotMatch
+        '404':
+          description: Not Found i.e. No Such Key
+  '/storageClasses':
```

Comment:
IMHO, need /v1 in the url.

Reply:
I am not sure about the versioning of APIs. Are we really maintaining api versions in the code?

Reply:
@himanshuvar is there any issues in keeping v1 in the API signature? could you please check?

Reply:
I just realized (discussed with Ashit as well), the v1 is not there in the code, right. This is just swagger updated based on the code...
@sfzeng can you please check and resolve this

Reply:
Yes, The Issue can be tracked  as below regarding versioning for API

Issue: https://github.com/sodafoundation/multi-cloud/issues/999
PR: https://github.com/sodafoundation/multi-cloud/pull/998